src/components/10-atoms/title-primary/index.js

```diff
@@ -0,0 +1,60 @@
+import { LitElement, html, css, unsafeCSS } from 'lit-element';
+import { classMap } from 'lit-html/directives/class-map';
+
+/* eslint-disable import/no-extraneous-dependencies */
+import defineOnce from '../../../utils/define-once';
+import styles from './index.scss';
+
+class AXATitlePrimary extends LitElement {
+  static get tagName() {
+    return 'axa-title-primary';
+  }
+
+  static get styles() {
+    return css`
+      ${unsafeCSS(styles)}
+    `;
+  }
+
+  static get properties() {
+    // Define properties and types
+    return {
+      variant: { type: String },
+    };
+  }
+
+  constructor() {
+    super();
+    this.variant = '';
+  }
+
+  render() {
+    // eslint-disable-next-line prefer-destructuring
+    const variant = this.variant;
+    const isSize1 = variant.includes('size-1');
+    const isSize2 = variant.includes('size-2');
+    const isSize3 = variant.includes('size-3');
+    const isSize4 = variant.includes('size-4');
+    const isSize5 = variant.includes('size-5');
+    const isSize6 = variant.includes('size-6');
```

Comment:
I would make the sized mutually exclusive, so that one text can't be all the sizes at once. A switch statement would be appropriate imho.

Reply:
I disagree. Variants are modifiers that can and should be able to be combined. Indeed for sizes, it does not make too much sense. Still, we should shot not prevent any "users" fault. We should show the modifiers that a user puts on it. Preventing it results also in even more unexpected behavior.